hitch-2011/hitch-fe
# Feature/pwa setup

## Is this a fix or a feature?
Feature
## What is the change?
PWA has been initialized
new logo added for downloads
## What does it fix?
n/a
## Where should the reviewer start?
review the new manifest. 
## How should this be tested?
Run a build and travel to local server. you should be prompted to download the app
## Relevant ScreenShots
